coretex/_task/process_utils.py

```diff
@@ -0,0 +1,85 @@
+from typing import List, Any
+from pathlib import Path
+from logging import Logger
+
+import subprocess
+
+from threading import Thread
+
+from . import LoggerUploadWorker
+from ..entities import Log, LogSeverity
+
+
+def _handleOutput(process: subprocess.Popen, worker: LoggerUploadWorker, logger: Logger) -> None:
+    stdout = process.stdout
+    if stdout is None:
+        raise ValueError("stdout is None for subprocess")
+
+    while process.poll() is None:
+        line: str = stdout.readline().decode("UTF-8")
+        if line.strip() == "":
+            continue
+
+        worker.add(Log.create(line.rstrip(), LogSeverity.info))
+        logger.info(line.rstrip())
```

Comment:
Use a separate task logger for logging output of the process for both stderr and stdout